cardano-cli/test/cardano-cli-golden/files/input/governance/costmodels.json

```diff
@@ -0,0 +1,573 @@
+      "unListData-memory-arguments": 1,
+      "unMapData-cpu-arguments": 1,
+      "unMapData-memory-arguments": 1,
+      "verifyEcdsaSecp256k1Signature-cpu-arguments": 1,
+      "verifyEcdsaSecp256k1Signature-memory-arguments": 1,
+      "verifyEd25519Signature-cpu-arguments-intercept": 1,
+      "verifyEd25519Signature-cpu-arguments-slope": 1,
+      "verifyEd25519Signature-memory-arguments": 1,
+      "verifySchnorrSecp256k1Signature-cpu-arguments-intercept": 1,
+      "verifySchnorrSecp256k1Signature-cpu-arguments-slope": 1,
+      "verifySchnorrSecp256k1Signature-memory-arguments": 1,
+      "cekConstrCost-exBudgetCPU": 1,
+      "cekConstrCost-exBudgetMemory": 1,
+      "cekCaseCost-exBudgetCPU": 1,
+      "cekCaseCost-exBudgetMemory": 1,
+      "bls12_381_G1_add-cpu-arguments": 1,
+      "bls12_381_G1_add-memory-arguments": 1,
+      "bls12_381_G1_compress-cpu-arguments": 1,
+      "bls12_381_G1_compress-memory-arguments": 1,
+      "bls12_381_G1_equal-cpu-arguments": 1,
+      "bls12_381_G1_equal-memory-arguments": 1,
+      "bls12_381_G1_hashToGroup-cpu-arguments-intercept": 1,
+      "bls12_381_G1_hashToGroup-cpu-arguments-slope": 1,
+      "bls12_381_G1_hashToGroup-memory-arguments": 1,
+      "bls12_381_G1_neg-cpu-arguments": 1,
+      "bls12_381_G1_neg-memory-arguments": 1,
+      "bls12_381_G1_scalarMul-cpu-arguments-intercept": 1,
+      "bls12_381_G1_scalarMul-cpu-arguments-slope": 1,
+      "bls12_381_G1_scalarMul-memory-arguments": 1,
+      "bls12_381_G1_uncompress-cpu-arguments": 1,
+      "bls12_381_G1_uncompress-memory-arguments": 1,
+      "bls12_381_G2_add-cpu-arguments": 1,
+      "bls12_381_G2_add-memory-arguments": 1,
+      "bls12_381_G2_compress-cpu-arguments": 1,
+      "bls12_381_G2_compress-memory-arguments": 1,
+      "bls12_381_G2_equal-cpu-arguments": 1,
+      "bls12_381_G2_equal-memory-arguments": 1,
+      "bls12_381_G2_hashToGroup-cpu-arguments-intercept": 1,
+      "bls12_381_G2_hashToGroup-cpu-arguments-slope": 1,
+      "bls12_381_G2_hashToGroup-memory-arguments": 1,
+      "bls12_381_G2_neg-cpu-arguments": 1,
+      "bls12_381_G2_neg-memory-arguments": 1,
+      "bls12_381_G2_scalarMul-cpu-arguments-intercept": 1,
+      "bls12_381_G2_scalarMul-cpu-arguments-slope": 1,
+      "bls12_381_G2_scalarMul-memory-arguments": 1,
+      "bls12_381_G2_uncompress-cpu-arguments": 1,
+      "bls12_381_G2_uncompress-memory-arguments": 1,
+      "bls12_381_finalVerify-cpu-arguments": 1,
+      "bls12_381_finalVerify-memory-arguments": 1,
+      "bls12_381_millerLoop-cpu-arguments": 1,
+      "bls12_381_millerLoop-memory-arguments": 1,
+      "bls12_381_mulMlResult-cpu-arguments": 1,
+      "bls12_381_mulMlResult-memory-arguments": 1,
+      "keccak_256-cpu-arguments-intercept": 1,
+      "keccak_256-cpu-arguments-slope": 1,
+      "keccak_256-memory-arguments": 1,
+      "blake2b_224-cpu-arguments-intercept": 1,
+      "blake2b_224-cpu-arguments-slope": 1,
+      "blake2b_224-memory-arguments": 1
+  }
+}
```

Comment:
Where did this file come from?

Reply:
Nice. Ledger can support the "old" cost model style :) https://github.com/input-output-hk/cardano-ledger/pull/3367/files#diff-f481d9eb9f793235d9fc4a1965ec1e0a01bed313f90adb11dcd9aa099a4cd738R348